### PR TITLE
Added more urls to BI list

### PR DESCRIPTION
**Pull request by Tafiti**
This list draws largely from alexa listing top 500 and a few from Burundians.

### CODE REVIEW

File: lists/bi.csv [hunk 1]
```diff
@@ -148,3 +148,354 @@ http://fuckbookafricaine.com,DATE,Online Dating,2017-03-02,CIPIT,
 http://soundcloud.com,MMED,Media sharing,2017-03-02,CIPIT,
 http://unzipper.com,FILE,File-sharing,2017-03-02,CIPIT,
 http://futura-sciences.com,HOST,Hosting and Blogging Platforms,2017-03-02,CIPIT,
+url,category_code,category_description,date_added,source,notes
```
Comment (hellais) [inline lists/bi.csv]: I think you added an extra header here.

**Comment by hellais** [comment]
It seems like there is still some issue with the list: https://travis-ci.org/citizenlab/test-lists/builds/210542168#L126.

There is an invalid category code on line 479

**Comment by hellais** [comment]
`+http://cours-gratuit.com,,,2017-03-09,CIPIT,`

Does not have a category code.

**Comment by hellais** [comment]
Ok it seems now the issue is that there is a duplicate URL (`DUPLICATE URL http://yaga-burundi.com on line 487 of lists/bi.csv`).

Sorry for reporting one problem at a time. This is going to be fixed in #141

**Comment by Tafiti** [comment]
Have you fixed it? I cannot seem to find the duplication on line #141 but I can see it on #487

-M

**Comment by hellais** [comment]
> Have you fixed it? I cannot seem to find the duplication on line #141 but I can see it on #487

Sorry what I meant to say is that in #141 I will fix the issue of not reporting all the problems with the test list on running the integration. That PR does not touch the test list itself though.

You will need to edit and remove the duplicate

**Comment by hellais** [comment]
> Have you fixed it? I cannot seem to find the duplication on line #141 but I can see it on #487

Sorry what I meant to say is that in #141 I will fix the issue of not reporting all the problems with the test list on running the integration. That PR does not touch the test list itself though.

You will need to edit and remove the duplicate

**Comment by hellais** [comment]
It seems like there is still one remaining issue with this PR:
`lists/bi.csv (line 499): Duplicate URL "http://rpa.bi"`

File: lists/bi.csv [hunk 2]
```diff
@@ -148,3 +148,352 @@ http://fuckbookafricaine.com,DATE,Online Dating,2017-03-02,CIPIT,
...
+http://www.anss.bi,PUBH,Public Health,2017-03-09,CIPIT,
+http://www.cnared.info/wordpress,POLR,Political Criticism,2017-03-09,CIPIT,
+http://www.rpa.bi/index.php/nos-journaux-parles/kirundi,CULTR,Culture,2017-03-09,CIPIT,
+http://www.rpa.bi/index.php/nos-journaux-parles/francais,CULTR,Culture,2017-03-09,CIPIT,
+http://www.rpa.bi/index.php/nos-journaux-parles/2011-08-15-07-19-47,CULTR,Culture,2017-03-09,CIPIT,
+https://www.facebook.com/sosmediasburundi,POLR,Political Criticism,2017-03-09,CIPIT,
+http://www.burundi.gov.bi,GOVT,Government,2017-03-09,CIPIT,
+http://burundinews.free.fr/actualites/actualites.htm,COMM,E-commerce,2017-03-09,CIPIT,
+http://www.inzamba.org/,CULTR,Culture,2017-03-09,CIPIT,
+
```
Comment (hellais) [inline lists/bi.csv]: Remove last empty line and it should be good to go.

**Comment by hellais** [comment]
Ok all green. Merging!

Thanks!

**Pull request merged**